packages/q/qtkeychain/pspec_x86_64.xml

```diff
@@ -1,42 +1,42 @@
 <PISI>
     <Source>
         <Name>qtkeychain</Name>
+        <Homepage>https://github.com/frankosterfeld/qtkeychain</Homepage>
         <Packager>
             <Name>Reilly Brogan</Name>
             <Email>solus@reillybrogan.com</Email>
         </Packager>
         <License>BSD-3-Clause</License>
-        <PartOf>security.library</PartOf>
+        <PartOf>debug</PartOf>
```

Comment:
Is this supposed to happen?

Reply:
As far as I can tell this is a bug in ypkg that happens when you have a package with subpackages however nothing ends up in the "main" package. It's harmless as the correct component is applied to all of the subpackages.